e2e/cypress/tests/integration/enterprise/overage_users/info_overage_users_spec.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {UserProfile} from 'mattermost-redux/types/users';
+
+const seatsPurchased = parseInt(Cypress.env('numberOfTrialUsers'), 10);
+const seatsMinimumFor5PercentageState = (Math.ceil(seatsPurchased * 0.05)) + seatsPurchased;
+const emailTest = 'test@mattermost.com';
+const text5PercentageState = `Your workspace user count has exceeded your paid license seat count by ${seatsMinimumFor5PercentageState - seatsPurchased} seats. Purchase additional seats to remain compliant.`;
+
+const withEmailLicense = (email = emailTest) => {
+    cy.intercept('GET', '**/api/v4/license/client', {
+        statusCode: 200,
+        body: {
+            IsLicensed: 'true',
+            IssuedAt: '1517714643650',
+            StartsAt: '1517714643650',
+            ExpiresAt: '1620335443650',
+            SkuShortName: 'Enterprise',
+            Name: 'LicenseName',
+            Company: 'Mattermost Inc.',
+            Users: String(seatsPurchased),
+            Email: email,
+        },
+    });
+};
+
+describe('Global Info Banner overage users', () => {
+    let url: string;
+    let createdUser: UserProfile;
+
+    beforeEach(() => {
+        cy.intercept('GET', '**/api/v4/analytics/old', {
+            statusCode: 200,
+            body: [
+                {name: 'channel_open_count', value: 2},
+                {name: 'channel_private_count', value: 0},
+                {name: 'post_count', value: 3},
+                {name: 'unique_user_count', value: seatsMinimumFor5PercentageState},
+                {name: 'team_count', value: 1},
+                {name: 'total_websocket_connections', value: 0},
+                {name: 'total_master_db_connections', value: 12},
+                {name: 'total_read_db_connections', value: 0},
+                {name: 'daily_active_users', value: 1},
+                {name: 'monthly_active_users', value: 1},
+                {name: 'inactive_user_count', value: 0},
+            ],
+        });
+
+        cy.apiInitSetup().then(({user, offTopicUrl}) => {
+            url = offTopicUrl;
+            createdUser = user;
+            cy.apiAdminLogin();
+            cy.apiDeleteLicense();
+            cy.visit(url);
+        });
+    });
+
+    it('should show the banner with CTA because it\'s the admin who purchased the license', () => {
```

Comment:
thank you for the tests @Afsoon .
- Do you think we can merge both files ? (It will be less load in splitting specs CI runners).
- in case we are using this `cy.apiDeleteLicense()`, can we keep it outside /cloud folder. It kinda blocks the other cloud tests that require license. I can help if you need anything.